WalletWasabi.Tests/UnitTests/Tor/Socks5/TorHttpPoolTests.cs

```diff
@@ -184,8 +184,8 @@ public TransportStream(string testName)
 			public async Task ConnectAsync(CancellationToken cancellationToken)
 			{
 				Task connectClientTask = Server.WaitForConnectionAsync(cancellationToken);
-				await Client.ConnectAsync(cancellationToken).ConfigureAwait(false);
```

Comment:
This is correct too because `ConnectAsync(...)` function is not an xunit test method (which has `[Fact]` or `[Theory]` attribute)